pkg/sql/schema_changer.go

```diff
@@ -778,10 +778,12 @@ func (sc *SchemaChanger) done(ctx context.Context) (*sqlbase.ImmutableTableDescr
 	}
 
 	var indexGCJobs []*jobs.StartableJob
+	var pkChangeIndexDropJob *jobs.StartableJob
 	update := func(txn *kv.Txn, descs map[sqlbase.ID]*sqlbase.MutableTableDescriptor) error {
 		// Reset vars here because update function can be called multiple times in a retry.
 		isRollback = false
 		indexGCJobs = nil
+		pkChangeIndexDropJob = nil
```

Comment:
Can we change this to a slice like "childJobs" or something so that it is easy to add new jobs to start immediately?